tests/test_bitwise.py

```diff
@@ -65,41 +67,47 @@ def test_bitwise_and(self, lhs, rhs, dtype):
         )
         assert_array_equal(dp_a & dp_b, np_a & np_b)
 
+        """
```

Comment:
please add a todo, that this will have to be uncommented once dpctl supports the use case with inplace bitwise operations